plugins/inputs/ceph/README.md

```diff
@@ -221,32 +257,53 @@ All admin measurements will have the following tags:
     - objects (float)
     - percent_used (float)
     - max_avail (float)
+    - stored (float)
 
 - ceph_pool_stats
   - tags:
     - name
   - fields:
     - read_bytes_sec (float)
     - write_bytes_sec (float)
-    - op_per_sec (float, exists only in ceph <10)
```

Comment:
Same remark here, either you sort the whole list, or keep this item at the same place.

Reply:
Latest changes have all fields sorted alphabetically.

Previously, they matched the ordering of what was already in the plugin, or the output of the `ceph` commands, but this is probably easier overall going forward.